pkg/plugin/builder/builder.go

```diff
@@ -66,7 +66,7 @@ func (b *builder) BuildPlugin() error {
 	}
 
 	// Build the .go file into a .so plugin
-	execErr := exec.Command("go", "build", "-buildmode=plugin", "-o", soFile, b.goFile).Run()
+	execErr := exec.Command("go", "build", "-mod=mod", "-buildmode=plugin", "-o", soFile, b.goFile).Run()
```

Comment:
Not a blocker -but hopefully this is redundant if all the plugins are also at 1.15 for plugins as well.

Reply:
So I actually ran into this during the `compose` step, and I think it has to do with the process creating a `vendor/` without a `modules.txt` before this happens (triggering an error `go: inconsistent vendoring in /Users/robertmulholand/go/src/github.com/makerdao/vdb-mcd-transformers`)